packages/time/src/unix-nano.ts

```diff
@@ -19,6 +19,10 @@ export class UnixNanoTimeStamp {
     this.input = input;
   }
 
+  public static newFn() {
+    return 1;
```

Comment:
?

Reply:
Oops that's me testing stuff, will remove